src/parser/jobs/smn/modules/GeneralCDDowntime.ts

```diff
@@ -1,38 +1,50 @@
 import ACTIONS from 'data/ACTIONS'
 import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
 
-const DEFAULT_ALLOWED_DOWNTIME = 4000
 // Although the trance cooldown is 55s, it is better in many cases
 // to hold for a 60s alignment, so some downtime per use must be allowed.
 const TRANCE_ALLOWED_DOWNTIME = 5000
 
 export default class GeneralCDDowntime extends CooldownDowntime {
-	defaultAllowedAverageDowntime = DEFAULT_ALLOWED_DOWNTIME
-	trackedCds = [
-		{
-			cooldowns: [ACTIONS.DREADWYRM_TRANCE, ACTIONS.FIREBIRD_TRANCE],
-			allowedAverageDowntime: TRANCE_ALLOWED_DOWNTIME,
-		},
-		{cooldowns: [
+	trackedCds = [ {
+		cooldowns: [
+			ACTIONS.DREADWYRM_TRANCE,
+			ACTIONS.FIREBIRD_TRANCE,
+		],
+		allowedAverageDowntime: TRANCE_ALLOWED_DOWNTIME,
+		firstUseOffset: 7500,
+	}, {
+		cooldowns: [
 			ACTIONS.ENERGY_DRAIN,
 			ACTIONS.ENERGY_SIPHON,
-		]},
-		{cooldowns: [
+		],
+		firstUseOffset: 2500,
+	}, {
+		cooldowns: [
 			ACTIONS.ASSAULT_I_AERIAL_SLASH,
 			ACTIONS.ASSAULT_I_EARTHEN_ARMOR,
 			ACTIONS.ASSAULT_I_CRIMSON_CYCLONE,
-		]},
-		{cooldowns: [
+		],
+		firstUseOffset: 1000,
+		// isAffectedBySpeed: true,
+	}, {
+		cooldowns: [
 			ACTIONS.ASSAULT_II_SLIIPSTREAM,
 			ACTIONS.ASSAULT_II_MOUNTAIN_BUSTER,
 			ACTIONS.ASSAULT_II_FLAMING_CRUSH,
-		]},
-		{cooldowns: [
+		],
+		firstUseOffset: 3500,
```

Comment:
Is it intentional to enforce the order of using Assault 1 then Assault 2 in neighboring weave slots, or would it also be acceptable to use them in the other order, such that both should have an allowable offset of 3.5s?

Reply:
I listed them in the order they are presented in the opener, but, no, there's no real reason you have to do one before the other.  I'll set them both to 3.5.